build.gradle

```diff
@@ -20,9 +24,9 @@ android {
     }
 
     // To let the libs be included in other projects
-    sourceSets {
```

Comment:
> The shared libraries for Android (armeabi, arm64-v8a, x86, ...) are already automatically included in the generated .aar files.

Great!

> 
However, the shared libraries for the host OS are not included in the .aar files or in any other .jar file. Should this change?

If it's possible to do include the host OS, sure please go ahead? That would help make things potentially easier for the users of the library.